.gitpod.welcome-message

```diff
@@ -0,0 +1,14 @@
+
+
+  _____ _ _   _____          _            _                               _
+ / ____(_) | |  __ \        | |   ___    | |                             | |
+| |  __ _| |_| |__) |__   __| |  ( _ )   | |     __ _ _ __ __ ___   _____| |
+| | |_ | | __|  ___/ _ \ / _` |  / _ \/\ | |    / _` | '__/ _` \ \ / / _ \ |
+| |__| | | |_| |  | (_) | (_| | | (_>  < | |___| (_| | | | (_| |\ V /  __/ |
+ \_____|_|\__|_|   \___/ \__,_|  \___/\/ |______\__,_|_|  \__,_| \_/ \___|_|
+
```

Comment:
Loved this 😍, can you please make it to `Gitpod & Laravel` (small `p` instead of `P`)

Reply:
Hi @Siddhant-K-code , glad you loved it!!  😍

Sure, I'll do that!